src/main/java/org/openpnp/model/BoardLocation.java

```diff
@@ -29,6 +31,8 @@
 public class BoardLocation extends AbstractModelObject {
 	@Element
 	private Location location;
+	@Element(required=false) // for compatility
```

Comment:
Is there a reason you put this on BoardLocation instead of Board? The dimensions of a Board are fixed to the board. Do you intend to use it for something else?